### DISCUSSION TITLE
stories_things/basic_content/this/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#8] [Announcements]
# stories_things/basic_content/this/

Stories About C Plus Plus

https://light-city.github.io/stories_things/basic_content/this/

**Comment by adamjiang2020** [comment]
楼主用的是什么软件调试C++？

**Reply by Light-City** [reply]
图中是Clion，你也可以用vscode，底层可以用gdb/lldb

**Comment by 1781559198** [comment]
想问问vscode怎么debug，我写完json老师报错
